src/Table.js

```diff
@@ -344,6 +344,7 @@ Table.defaultProps = {
   bodyRef: null,
   scrollRef: null,
   tableRef: null,
+  initialSortState: {},
```

Comment:
could we change this to `{ criteria: null, direction: null}` so they aren't `undefined`?

Reply:
Sure